src/blockchain-utilities/derivation.test.ts

```diff
@@ -83,11 +84,27 @@ describe('deriveWalletPathFromSponsorAddress', () => {
     it('throws deriving a sponsor wallet due to an invalid DApi name', () => {
       const dapiName = 'invalid dapi name';
       const throwingFn = () =>
-        deriveSponsorWallet('test test test test test test test test test test test junk', dapiName);
+        deriveSponsorWallet('test test test test test test test test test test test junk', dapiName, 'AIRSEEKER');
```

Comment:
I am not a huge fan of these uppercased protocol names, but don't want users to pass numbers. If this is lowercased, so should be `PROTOCOL_IDS` below.

Reply:
To me, uppercase suggests they represent more than any old string, which is a benefit. Uppercase also matches Airnode:

https://github.com/api3dao/airnode/blob/707491e6a54d94dfbda56f26d4be2e7504a945e4/packages/airnode-protocol/src/index.ts#L39-L46

Reply:
Ok. Leaving it as is then.

Reply:
I realized that we would be mixing `protocol ID` and `protocol name` in some functions, so I sticked with using the protocol ID everywhere.